src/arena/cfr/export.rs

```diff
@@ -0,0 +1,488 @@
+        writeln!(
+            file,
+            "  node_{} [label=\"{}\", shape={}, style=\"filled\", fillcolor=\"{}\"];",
+            node.idx, label, shape, color
+        )?;
+        
+        // Process edges to children using the new iter_children method
+        for (child_idx, child_node_idx) in node.iter_children() {
+            let edge_label = match &node.data {
+                NodeData::Chance => {
+                    // For chance nodes, label the edge with the card
+                    let card = Card::try_from(child_idx as u8).map_or_else(
+                        |_| format!("Card {}", child_idx),
+                        |card| format!("{}", card)
+                    );
+                    card
+                },
+                NodeData::Player(_) => {
+                    // For player nodes, label the action
+                    // Index 0 is fold according to the issue description
+                    if child_idx == 0 {
+                        "Fold".to_string()
+                    } else if child_idx == 1 {
+                        "Check/Call".to_string()
+                    } else {
+                        format!("Bet/Raise {}", child_idx - 1)
+                    }
+                },
+                _ => format!("{}", child_idx),
+            };
+            
+            // Use the count for this edge to indicate frequency
+            let count = node.get_count(child_idx);
+            let edge_style = if count > 0 {
+                format!(" [label=\"{} (count: {})\", penwidth={}]", edge_label, count, 1.0 + (count as f32).min(5.0))
+            } else {
+                format!(" [label=\"{}\"]", edge_label)
+            };
+            
+            writeln!(
+                file,
+                "  node_{} -> node_{}{}",
+                node.idx, child_node_idx, edge_style
+            )?;
+        }
+    }
+    
+    // Close the graph
+    writeln!(file, "}}")?;
+    
+    Ok(())
+}
+
+/// Exports the CFR state tree to a PNG image using Graphviz.
+/// 
+/// This function creates a DOT file and then converts it to PNG using the 
+/// Graphviz dot utility. The dot utility must be installed on the system.
+/// 
+/// # Arguments
+/// 
+/// * `state` - A reference to the CFR state to export
+/// * `output_path` - The path where the PNG file will be saved
+/// * `cleanup_dot` - Whether to remove the temporary DOT file after conversion
+/// 
+/// # Returns
+/// 
+/// * `io::Result<()>` - Success or error
+pub fn export_to_png(state: &CFRState, output_path: &Path, cleanup_dot: bool) -> io::Result<()> {
+    // Create a temporary DOT file
+    let dot_path = output_path.with_extension("dot");
+    
+    // Generate DOT file
+    export_to_dot(state, &dot_path)?;
+    
+    // Use Graphviz to convert DOT to PNG
+    let status = std::process::Command::new("dot")
+        .arg("-Tpng")
+        .arg(&dot_path)
+        .arg("-o")
+        .arg(output_path)
+        .status()?;
```

Comment:
We shouldn't require that users have dot installed.
https://crates.io/crates/graphviz-rust